### PR TITLE
Load Time optimization

### PR DESCRIPTION
**Pull request by zo-el**
- Update to  use `cache-and-network` instead of `network-only` for a better user experience

- New Query for `Home` page to get Counterparties  because it only needs Counterparties for the Completed transaction
> Note: We need to better optimize the issues to get counterparties for every page.

### CODE REVIEW

File: src/graphql-server/schema.graphql [hunk 1]
```diff
@@ -121,21 +121,22 @@ type Query {
   holofuelLedger: Ledger
   holofuelUser: HolofuelUser
   holofuelCounterparty(agentId: String!): HolofuelUser
+  holofuelHomeCounterparties: [HolofuelUser]
```
Comment (JettTech) [inline src/graphql-server/schema.graphql:124]: Great addition!  :clap:

File: src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js [hunk 2]
```diff
@@ -18,9 +18,9 @@ export function PrimaryLayout ({
   children,
   headerProps = {}
 }) {
-  const { data: { holofuelActionableTransactions: actionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'network-only' })
+  const { data: { holofuelActionableTransactions: actionableTransactions = [] } = {} } = useQuery(HolofuelActionableTransactionsQuery, { fetchPolicy: 'cache-and-network' })
```
Comment (JettTech) [inline src/holofuel/components/layout/PrimaryLayout/PrimaryLayout.js:21]: Not quite sure what changed, such that this fetchPolicy now works, but I'm *very happy it does*!

**Review by JettTech — APPROVED** [review]
Ship it! :ship: :package: :rocket:

**Comment by robbiecarlton** [comment]
Looks good!

**Review by robbiecarlton — APPROVED** [review]

**Pull request merged**